Dockerfile

```diff
@@ -1,31 +1,29 @@
-FROM phusion/baseimage:0.9.16
-
-RUN DEBIAN_FRONTEND=noninteractive apt-get update
-RUN DEBIAN_FRONTEND=noninteractive apt-get install -y apache2 supervisor
-RUN DEBIAN_FRONTEND=noninteractive apt-get install -y libapache2-mod-php5 php5-mysql
-
-RUN mkdir -p /var/lock/apache2 /var/run/apache2 /var/log/supervisor
-
-RUN php5enmod mysql
-
-RUN a2enmod php5 rewrite
-
-COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
-
-ADD vhost.conf /etc/apache2/sites-enabled/000-default.conf
-
-RUN mkdir /yourls
-
-RUN curl -L https://github.com/YOURLS/YOURLS/archive/1.7.1.tar.gz | tar -zx -C /yourls --strip-components=1
-
-ADD config.php /yourls/user/config.php
-
-ADD htaccess /yourls/.htaccess
-ADD migrate.php /yourls/migrate.php
-
-WORKDIR /yourls
-
-RUN chown -R www-data:www-data /yourls
+FROM ubuntu:14.04
+
+ENV APP_ROOT /yourls
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN mkdir -p ${APP_ROOT} \
```

Comment:
oh wait there it is, on line 10

he does all his apt stuff and then removes the apt cache on one `RUN` command so the layer is smaller. if the cache files get added in one layer and removed in a subsequent layer then the files are still in the resulting image because of the way docker manages the layers